spec/rails_helper.rb

```diff
@@ -1,9 +1,8 @@
 ENV['RACK_ENV'] = 'test'
 
 require 'spec_helper'
-
 require File.expand_path('../../config/environment', __FILE__)
-abort('DATABASE_URL environment variable is set') if ENV['DATABASE_URL']
+abort('DATABASE_URL environment variable is set') if Figaro.env.database_url
 
```

Comment:
tbh I'm not sure why this was ever here. Smells like it derived from a template or cargo culting.